liferay-workspace/modules/blade.modellistener/README.markdown

```diff
@@ -0,0 +1,51 @@
+# Model Listener
+
+## What does this sample do when it's deployed?
+
+The Model Listener sample demonstrates adding a custom model listener to a Liferay Portal's out of the box entity. When deploying this sample with no customizations, a custom model listener will be added to `layouts`, listening for `onBeforeCreate` events. This means that any page creation will trigger this listener, which will execute before the new page being created.
```

Comment:
@fafonso I'm curious about the following statement:

> a custom model listener will be added to `layouts`

When you refer to layouts, do you just mean the portal instance's layouts, in general? If so, I'd like to say something more like

> a custom model listener will be added to the portal's layouts

I see you have the term *layouts* in code format though, so I wanted to check to see if you meant something else before changing it. Thanks!